pydocstringformatter/formatting/__init__.py

```diff
@@ -1,11 +1,14 @@
-__all__ = ["FORMATTERS"]
+__all__ = ["FORMATTERS", "Formatter", "StringFormatter"]
```

Comment:
I made the Base class public here @DanielNoord

Reply:
Perhaps we should wait with that for `1.0.0`? Make sure our internal APIs are all stable and then decide what we want and what not to expose?

Reply:
Done